### PR TITLE
Create 2023-04-24-community-recognition.md

### PR DESCRIPTION
**Pull request by elletjies**
Please delete this line and the text below before submitting your contribution.

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.

---

### CODE REVIEW

File: _posts/2023/04/2023-04-24-community-recognition.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Last week our Team Leads and Executive Council met to discuss progress towards our strategic plan for 2020-2025. The last update we shared with the 
+community was in May of 2022. If you missed it, read our 
+[Strategic Plan Year Two in Review](https://carpentries.org/blog/2022/05/year-two-strategic-plan-update/). 
+During the hybrid meeting, our Team Leads shared significant milestones we have achieved in the last year related to the following six strategic 
+goals:
+
+1. Build regional and local capacity to empower sustainable communities
+2. Intentionally incorporate equity, inclusion, and accessibility to support a diverse community.
+3, Provide opportunities for our volunteers and Core Team's growth and professional development.
```
Comment (ErinBecker) [inline _posts/2023/04/2023-04-24-community-recognition.md]:
```suggestion
3, Provide opportunities for our volunteers' and Core Team's growth and professional development.
```

File: _posts/2023/04/2023-04-24-community-recognition.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+Each of these goals carries its specific objectives and rationale to aid us in implementing them.
+In our third year of work toward the Strategic Plan, we have made significant progress on each of the six goals. Read on to learn about our work on 
+each goal, and follow the [Strategic Plan GitHub repository](https://github.com/carpentries/strategic-plan) for quarterly updates.
+
+## Goal 1: Build regional and local capacity to empower sustainable communities
+
+For us to empower our regional and local communities, we need to know who leads those efforts. As part of our Community Development Program, we opened 
+a [subcommunity registry](https://carpentries.org/blog/2023/02/register-your-subcommunity/). By registering your sub-community, help us raise awareness 
+about the wonderful Carpentries activities happening worldwide. Your registration will also help guide how we recognise and appreciate you for your work 
+to advance our mission. Are you following the Carpentries activities across Africa? Check out this
```
Comment (ErinBecker) [inline _posts/2023/04/2023-04-24-community-recognition.md]:
```suggestion
in advancing our mission. Are you following the Carpentries activities across Africa? Check out this 
```

File: _posts/2023/04/2023-04-24-community-recognition.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+a [subcommunity registry](https://carpentries.org/blog/2023/02/register-your-subcommunity/). By registering your sub-community, help us raise awareness 
+about the wonderful Carpentries activities happening worldwide. Your registration will also help guide how we recognise and appreciate you for your work 
+to advance our mission. Are you following the Carpentries activities across Africa? Check out this 
+[retrospective 2022](https://docs.google.com/presentation/d/1p2t_kxMN8GWV_ddZpgOENVq7OAns_nD8lGq5hJu_xIk/edit#slide=id.g102fef8a4b2_0_14) 
+highlighting Glosario, monthly meetups, and other achievements to build regional capacity in our African communities.
+
+## Goal 2: Intentionally incorporate equity, inclusion, and accessibility to support a diverse community.
+
+One of the ways we are incorporating equity, inclusion, and accessibility is by making the Instructor checkout process more accessible and inclusive 
+across our global community (see [recommended changes](https://github.com/carpentries/trainers/issues/226)). We have also designated funds in our fiscal
```
Comment (ErinBecker) [inline _posts/2023/04/2023-04-24-community-recognition.md]:
```suggestion
across our global community (see [changes we are working to implement](https://github.com/carpentries/trainers/issues/226)). We have also designated funds in our fiscal 
```

File: _posts/2023/04/2023-04-24-community-recognition.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+highlighting Glosario, monthly meetups, and other achievements to build regional capacity in our African communities.
+
+## Goal 2: Intentionally incorporate equity, inclusion, and accessibility to support a diverse community.
+
+One of the ways we are incorporating equity, inclusion, and accessibility is by making the Instructor checkout process more accessible and inclusive 
+across our global community (see [recommended changes](https://github.com/carpentries/trainers/issues/226)). We have also designated funds in our fiscal 
+year budget to translate key resources like the Code of Conduct. We are in our final round of functionality updates for our website (www.carpentries.org) and will soon move into community review and launch. And finally, in an effort to improve the accessibility of our workshops and community spaces, we developed an [Accessibility Accommodation Form](https://carpentries.typeform.com/to/B2OSYaD0) and released an [Accessibility Statement](https://carpentries.org/blog/2023/03/accessibility-statement-and-accommodation-form/).
```
Comment (ErinBecker) [inline _posts/2023/04/2023-04-24-community-recognition.md]:
```suggestion
year budget to translate key resources like the Code of Conduct. We are in our final round of functionality updates for our website and will soon move into community review and launch. And finally, in an effort to improve the accessibility of our workshops and community spaces, we developed an [Accessibility Accommodation Form](https://carpentries.typeform.com/to/B2OSYaD0) and released an [Accessibility Statement](https://carpentries.org/blog/2023/03/accessibility-statement-and-accommodation-form/). 
```

Comment (ErinBecker) [inline _posts/2023/04/2023-04-24-community-recognition.md]:
```suggestion
year budget to translate key resources like the Code of Conduct. We are in our final round of functionality updates for our website (www.carpentries.org) and will soon move into community review and launch. In May, we will launch our new lesson infrastructure, [The Carpentries Workbench](https://carpentries.github.io/workbench/), which prioritises accessibility. And finally, in an effort to improve the accessibility of our workshops and community spaces, we developed an [Accessibility Accommodation Form](https://carpentries.typeform.com/to/B2OSYaD0) and released an [Accessibility Statement](https://carpentries.org/blog/2023/03/accessibility-statement-and-accommodation-form/). 
```

File: _posts/2023/04/2023-04-24-community-recognition.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+One of the ways we are incorporating equity, inclusion, and accessibility is by making the Instructor checkout process more accessible and inclusive 
+across our global community (see [recommended changes](https://github.com/carpentries/trainers/issues/226)). We have also designated funds in our fiscal 
+year budget to translate key resources like the Code of Conduct. We are in our final round of functionality updates for our website (www.carpentries.org) and will soon move into community review and launch. And finally, in an effort to improve the accessibility of our workshops and community spaces, we developed an [Accessibility Accommodation Form](https://carpentries.typeform.com/to/B2OSYaD0) and released an [Accessibility Statement](https://carpentries.org/blog/2023/03/accessibility-statement-and-accommodation-form/). 
+
+## Goal 3: Provide opportunities for growth and professional development of our volunteers and Core Team.
+
+We learned so much implementing CarpentryCon, including how beneficial skill-up sessions are for our community. We are excited to share that we’ll
```
Comment (ErinBecker) [inline _posts/2023/04/2023-04-24-community-recognition.md]:
```suggestion
We learned so much implementing CarpentryCon 2022, including how beneficial skill-up sessions are for our community. We are excited to share that we’ll 
```

File: _posts/2023/04/2023-04-24-community-recognition.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+year budget to translate key resources like the Code of Conduct. We are in our final round of functionality updates for our website (www.carpentries.org) and will soon move into community review and launch. And finally, in an effort to improve the accessibility of our workshops and community spaces, we developed an [Accessibility Accommodation Form](https://carpentries.typeform.com/to/B2OSYaD0) and released an [Accessibility Statement](https://carpentries.org/blog/2023/03/accessibility-statement-and-accommodation-form/). 
+
+## Goal 3: Provide opportunities for growth and professional development of our volunteers and Core Team.
+
+We learned so much implementing CarpentryCon, including how beneficial skill-up sessions are for our community. We are excited to share that we’ll 
+provide ongoing programming similar to what you’d see at CarpentryCon, throughout the year to develop our community members’ skills in teaching, lesson 
+development, and community building. We also launched a Supervisor Pilot Program for Core Team members interested in growing their supervisory skills.
```
Comment (ErinBecker) [inline _posts/2023/04/2023-04-24-community-recognition.md]:
```suggestion
development, and community building. Within the Core Team, we also launched a Supervisor Pilot Program for Core Team members interested in growing their supervisory skills.
```

File: _posts/2023/04/2023-04-24-community-recognition.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+## Goal 4: Facilitate the creation and maintenance of relevant, high-quality, community-curated curricula.
+
+Our new lesson infrastructure is to be rolled out on 1 May 2023, redesigned with a focus on accessibility and ease of maintenance. Additionally, we are
```
Comment (ErinBecker) [inline _posts/2023/04/2023-04-24-community-recognition.md]:
```suggestion
[The Carpentries Workbench](https://carpentries.github.io/workbench/), to be rolled out in May, greatly simplifies the process of contributing to lessons. Additionally, we are 
```

File: _posts/2023/04/2023-04-24-community-recognition.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+We learned so much implementing CarpentryCon, including how beneficial skill-up sessions are for our community. We are excited to share that we’ll 
+provide ongoing programming similar to what you’d see at CarpentryCon, throughout the year to develop our community members’ skills in teaching, lesson 
+development, and community building. We also launched a Supervisor Pilot Program for Core Team members interested in growing their supervisory skills.
+
+## Goal 4: Facilitate the creation and maintenance of relevant, high-quality, community-curated curricula.
+
+Our new lesson infrastructure is to be rolled out on 1 May 2023, redesigned with a focus on accessibility and ease of maintenance. Additionally, we are 
+piloting a new curriculum for Collaborative Lesson Development Training, meeting demand from the community. We’ve also updated and expanded our 
+Maintainer onboarding program to support this essential role in the community better and established Curriculum Advisory Committees to guide the development of all official curricula.
```
Comment (ErinBecker) [inline _posts/2023/04/2023-04-24-community-recognition.md]:
```suggestion
Maintainer onboarding program to support this essential role in the community better, and established Curriculum Advisory Committees to guide the development of all official curricula.
```

File: _posts/2023/04/2023-04-24-community-recognition.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+We have also received approval on our trademarks and appreciate the community feedback on our logo policy. We’ve made updates to core data privacy and 
+security practices, including our Privacy Policy and online accounts safety. To strengthen our governance, we have onboarded governance committees for 
+all official lesson programs. These committees are responsible for overseeing the strategy of their lesson programs, advocating for it, and communicating with Executive Council, Core Team, and community.
+Routine emails to existing Members, including benefit usage reports, [are now automated](https://github.com/carpentries/strategic-plan/issues/55#issuecomment-1160757632), reducing administration time for this program. And lastly, we have implemented a simplified process for Memberships with Trainers.
+
+## Goal 6: Advocate for our values and vision to empower more people to work with data and software.
+
+We are proud to announce we are coordinating with the HPC Carpentry community on their application for lesson program incubation. Additionally, we have
```
Comment (ErinBecker) [inline _posts/2023/04/2023-04-24-community-recognition.md]:
```suggestion
We are proud to announce we are coordinating with the [HPC Carpentry](https://www.hpc-carpentry.org/) community on their application to become an official lesson program within The Carpentries. Additionally, we have 
```

File: _posts/2023/04/2023-04-24-community-recognition.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+
+## Goal 6: Advocate for our values and vision to empower more people to work with data and software.
+
+We are proud to announce we are coordinating with the HPC Carpentry community on their application for lesson program incubation. Additionally, we have 
+presented lesson infrastructure and lesson development pathways at conferences and other events. We are collaborating with several partners, including 
+[2i2c](https://2i2c.org/) and [MetaDocencia](https://www.metadocencia.org/) on a CZI-funded project to empower research communities in the Global South 
+to use open science cloud technologies. Lastly, several of our Core Team and community members serve on advisory boards of other organisations to provide guidance and advocacy in areas key to The Carpentries mission and vision. Examples include the Education Committee for Ecological Society of America, AG2PI Scientific Advisory Board, and NASA’s Transform to Open Science (TOPS) panel.
+
+## What is next?
+We will continue to monitor and improve services, tools and virtual infrastructure to achieve reliability, efficiency and safety and security for our
```
Comment (ErinBecker) [inline _posts/2023/04/2023-04-24-community-recognition.md]:
```suggestion
We will continue to monitor and improve services, tools and virtual infrastructure to enhance reliability, efficiency and safety and security for our 
```

**Review by ErinBecker — CHANGES_REQUESTED** [review]
Thanks @elletjies - this is looking great! I've made some suggested changes.

**Comment by elletjies** [comment]
Thank you @ErinBecker!

**Pull request merged**